library/asn1parse.c

```diff
@@ -98,6 +98,11 @@ int mbedtls_asn1_get_len(unsigned char **p,
     return 0;
 }
 
+static int mbedtls_asn1_bad_tag(int tag)
```

Comment:
In this library, as you note, a tag is always a single byte.
```suggestion
static int mbedtls_asn1_bad_tag(unsigned char tag)
```